ofrak_core/test_ofrak/components/test_filesystem_component.py

```diff
@@ -397,3 +407,25 @@ def diff_directories(dir_1, dir_2, extra_diff_flags):
             second_type = " ".join(second.split(" ")[4:])
 
             assert first_type == second_type
+
+
+class TestCPIOFilesystem:
+    async def test_unpack(self, cpio_filesystem_root):
+        await cpio_filesystem_root.unpack()
+
+    async def test_absolute_paths(self, cpio_filesystem_root):
+        await cpio_filesystem_root.unpack()
+        # Check that absolute path '/dev/console' exists after unpacking to
+        # ensure that files in filesystems with absolute paths unpack
+        # successfully without overwriting system files
+        children = list(await cpio_filesystem_root.get_children_as_view(FilesystemEntry))
+        children_names = [child.get_name() for child in children]
+        assert "dev" in children_names
+        dev = await cpio_filesystem_root.get_only_child(
+            r_filter=ResourceFilter(
+                attribute_filters=(ResourceAttributeValueFilter(FilesystemEntry.Name, "dev"),)
+            )
+        )
+        grandchildren = list(await dev.get_children_as_view(FilesystemEntry))
+        grandkid_names = [grandkid.get_name() for grandkid in grandchildren]
+        assert "console" in grandkid_names
```

Comment:
could all of this be rewritten with something like:
```suggestion
        filesystem_view = await cpio_filesystem_root.view_as(FilesystemRoot)
        dev_console = await cpio_filesystem.get_entry("dev/console")
        assert dev_console is not None
```